docs/plugins/web.rst

```diff
@@ -6,6 +6,11 @@ supplements the CLI. It can't do much right now, and the interface is a little
 clunky, but you can use it to query and browse your music and---in browsers that
 support HTML5 Audio---you can even play music.
 
+It also implements a subset of `subsonic api`_ that allows any subsonic client
+to play music from beets.
+
+.. _subsonic api: http://www.subsonic.org/pages/api.jsp
```

Comment:
The docs should probably list the required dependencies (whether we use a separate plugin or expand the current web plugin).